### DISCUSSION TITLE
Whole organism part name - needs discussion

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4833] [General]
Please read through #4758 

Our current use of and definition for the part name whole organism is not being consistently applied.

[whole organism](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#whole_organism) - A contiguous alive or once living physical entity; entity or being that is or was living; an individual living or once living thing, such as one animal, plant, fungus, or bacterium. [Wikidata](https://www.wikidata.org/wiki/Q7239)

Issues to note are:

1. Herbaria use "whole organism" for almost anything that is on a herbarium sheet, whether it is an entire plant or a branch with some leaves. There may be 5 sheets from the same plant in a single catalog record, so while the count of "whole organism" parts = 5, the individual count = 1
2. @DerekSikes says - Some people think the key word is 'whole' as in 'undivided' whereas I think of it as meaning "the majority of the organism" so that if we take a leg off an insect and put it in a cryovial we'll have two parts: whole organism on a pin, leg in a vial.
3. In most vertebrate collections, there is no whole organism part, just skin and skeleton or something like that. (Although it was the "whole organism" that was collected)
4. It has been observed in may philosophical discussions of the TDWG Material;Sample Task Group that one can never really have a whole organism. An organism is more like a concept that describes a living thing from conception to dissolution via decay. Please read https://github.com/tdwg/material-sample/issues/2#issuecomment-945976817 and also https://github.com/tdwg/material-sample/issues/2#issuecomment-947796673 (and everything in between if you have a lot of time on your hands...)

So while Derek sees denormalization in creating both a whole organism with lot count of 1 and an individual count attribute of 1, I think the issue is really that we have confusing terms and definitions.

Using "whole organism" as a part name is not describing what we actually have for the majority of cases in which it is used (actually all, if you consider the organism to be what is described in 4 above).

Our definition for individual count is describing organisms NOT parts. Do we need to make that more clear?

[individual count](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#individual_count) - The number of individual organisms represented by the catalog record. [Github](https://github.com/ArctosDB/arctos/issues/4032)

To Derek's point, should we have something other than "whole organism" to appropriately describe the insect missing a leg, the fish in a jar missing a fin, or the herbarium sheet with only a branch, leaves, and flower on it? Should we even use this term as a part name at all?

Let the philosophizing begin....

Also - note that individual count is what shows up at aggregators in [dwc:individualCount](https://dwc.tdwg.org/terms/#dwc:individualCount) and is required for proper publication of DMNS:Inv data to OBIS. As far as I am aware, no other aggregators require this information, but given that this is the only place to precisely say - this record applies to x number of individual organisms - it would probably be good for all the biological collections to make good use of it.

**Comment by DerekSikes** [comment]
Maybe an improvement for entomology (insect on a pin) would be to use
"body" instead of "whole organism." This removes the "whole" part which
makes it more generalizable so it could be used for an insect on a pin that
is intact, or one that's had a leg removed into a vial (or one that's just
damaged, eg missing its head).

I think that term is intuitive enough that most people would think it means
"all or most of an organism"

But it's probably not that useful for plants?

On Thu, Jun 16, 2022 at 7:01 AM Teresa Mayfield-Meyer <
***@***.***> wrote:

> Please read through #4758 <https://github.com/ArctosDB/arctos/issues/4758>
>
> Our current use of and definition for the part name whole organism is not
> being consistently applied.
>
> whole organism
> <https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#whole_organism>
> - A contiguous alive or once living physical entity; entity or being that
> is or was living; an individual living or once living thing, such as one
> animal, plant, fungus, or bacterium. Wikidata
> <https://www.wikidata.org/wiki/Q7239>
>
> Issues to note are:
>
>    1. Herbaria use "whole organism" for almost anything that is on a
>    herbarium sheet, whether it is an entire plant or a branch with some
>    leaves. There may be 5 sheets from the same plant in a single catalog
>    record, so while the count of "whole organism" parts = 5, the individual
>    count = 1
>    2. @DerekSikes <https://github.com/DerekSikes> says - Some people
>    think the key word is 'whole' as in 'undivided' whereas I think of it as
>    meaning "the majority of the organism" so that if we take a leg off an
>    insect and put it in a cryovial we'll have two parts: whole organism on a
>    pin, leg in a vial.
>    3. In most vertebrate collections, there is no whole organism part,
>    just skin and skeleton or something like that. (Although it was the "whole
>    organism" that was collected)
>    4. It has been observed in may philosophical discussions of the TDWG
>    Material;Sample Task Group that one can never really have a whole organism.
>    An organism is more like a concept that describes a living thing from
>    conception to dissolution via decay. Please read tdwg/material-sample#2
>    (comment)
>    <https://github.com/tdwg/material-sample/issues/2#issuecomment-945976817>
>    and also tdwg/material-sample#2 (comment)
>    <https://github.com/tdwg/material-sample/issues/2#issuecomment-947796673>
>    (and everything in between if you have a lot of time on your hands...)
>
> So while Derek sees denormalization in creating both a whole organism with
> lot count of 1 and an individual count attribute of 1, I think the issue is
> really that we have confusing terms and definitions.
>
> Using "whole organism" as a part name is not describing what we actually
> have for the majority of cases in which it is used (actually all, if you
> consider the organism to be what is described in 4 above).
>
> Our definition for individual count is describing organisms NOT parts. Do
> we need to make that more clear?
>
> individual count
> <https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#individual_count>
> - The number of individual organisms represented by the catalog record.
> Github <https://github.com/ArctosDB/arctos/issues/4032>
>
> To Derek's point, should we have something other than "whole organism" to
> appropriately describe the insect missing a leg, the fish in a jar missing
> a fin, or the herbarium sheet with only a branch, leaves, and flower on it?
> Should we even use this term as a part name at all?
>
> Let the philosophizing begin....
>
> Also - note that individual count is what shows up at aggregators in
> dwc:individualCount <https://dwc.tdwg.org/terms/#dwc:individualCount> and
> is required for proper publication of DMNS:Inv data to OBIS. As far as I am
> aware, no other aggregators require this information, but given that this
> is the only place to precisely say - this record applies to x number of
> individual organisms - it would probably be good for all the biological
> collections to make good use of it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/4771>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM2KRJNDWY54CIERBI3VPM6TBANCNFSM5Y7DLCRA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 357,704 digitized arthropod records
<http://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by dustymc** [comment]
>Should we even use this term as a part name at all?

I don't have a problem with it. I'm not a fan of the wiki definition, but I don't really have a better one to offer either. (We know it when we see it...) I'm not sure it's ever quite correct, even barring the always-extreme TDWG viewpoint, but it's a handy, accessible proxy to "most of the thing." 

Moving "whole" to some part attribute would be a bit more correct and a bit less accessible - I don't see any reason to force that at this time.

The wiki definition is blocking any chance of getting a better one again, but I'm relatively certain that [body](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#body) is not intended to be interchangeable with "whole organism" - it's something about being skinned or something, and I think somehow differs from [carcass](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#carcass), which is maybe-similar and also has an uninformative definition. I suppose we should do something drastic with https://github.com/ArctosDB/arctos/issues/4368 if nobody wants to defend any of that.

>most people would think

Users should not have to guess at our terminology. We have documentation, it should be read and understood by both people entering data and those trying to find stuff. That's not to say intuitive terms aren't awesome, but everyone should have the ability to KNOW what we mean by any term at any time.

**Comment by DerekSikes** [comment]
Ok, how about the term 'individual' to replace 'whole organism'?

-Derek

On Thu, Jun 16, 2022 at 7:29 AM dustymc ***@***.***> wrote:

> Should we even use this term as a part name at all?
>
> I don't have a problem with it. I'm not a fan of the wiki definition, but
> I don't really have a better one to offer either. (We know it when we see
> it...) I'm not sure it's ever quite correct, even barring the
> always-extreme TDWG viewpoint, but it's a handy, accessible proxy to "most
> of the thing."
>
> Moving "whole" to some part attribute would be a bit more correct and a
> bit less accessible - I don't see any reason to force that at this time.
>
> The wiki definition is blocking any chance of getting a better one again,
> but I'm relatively certain that body
> <https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#body>
> is not intended to be interchangeable with "whole organism" - it's
> something about being skinned or something, and I think somehow differs
> from carcass
> <https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#carcass>,
> which is maybe-similar and also has an uninformative definition. I suppose
> we should do something drastic with #4368
> <https://github.com/ArctosDB/arctos/issues/4368> if nobody wants to
> defend any of that.
>
> most people would think
>
> Users should not have to guess at our terminology. We have documentation,
> it should be read and understood by both people entering data and those
> trying to find stuff. That's not to say intuitive terms aren't awesome, but
> everyone should have the ability to KNOW what we mean by any term at any
> time.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/4771#issuecomment-1157796713>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACFNUM5JARQACOWQIRNEXB3VPNB35ANCNFSM5Y7DLCRA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

+++++++++++++++++++++++++++++++++++
*Derek S. Sikes*, Curator of Insects, Professor of Entomology
University of Alaska Museum (UAM), University of Alaska Fairbanks
1962 Yukon Drive, Fairbanks, AK   99775-6960
***@***.*** phone: 907-474-6278 he/him/his
University of Alaska Museum <https://www.uaf.edu/museum/collections/ento/>
-  search 357,704 digitized arthropod records
<http://arctos.database.museum/uam_ento>
+++++++++++++++++++++++++++++++++++

Interested in Alaskan Entomology? Join the Alaska Entomological
Society and / or sign up for the email listserv "Alaska Entomological
Network" at
http://www.akentsoc.org/contact_us

**Comment by dustymc** [comment]
>individual

If we're changing, it would be nice to find something that works better for clonal organisms and such. Maybe ``body``, but I'm not sure how the botanists might feel about that, nor what the current intention is. I'll start another Issue for that....

Something that incorporates [object](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#object) would be even better, but maybe that's too far.
